specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/dedicatedHost.json

```diff
@@ -785,6 +791,15 @@
         "supportAutomaticPlacement": {
           "type": "boolean",
           "description": "Specifies whether virtual machines or virtual machine scale sets can be placed automatically on the dedicated host group. Automatic placement means resources are allocated on dedicated hosts, that are chosen by Azure, under the dedicated host group. The value is defaulted to 'false' when not provided. <br><br>Minimum api-version: 2020-06-01."
+        },
+        "additionalCapabilities": {
+          "properties": {
+            "ultraSSDEnabled": {
+              "type": "boolean",
+              "description": "The flag that enables or disables a capability to have UltraSSD Enabled Virtual Machines on Dedicated Hosts of the Dedicated Host Group. For the Virtual Machines to be UltraSSD Enabled, UltraSSDEnabled flag for the resoure needs to be set true as well. The value is defaulted to 'false' when not provided."
```

Comment:
I forgot to mention this before, but we need to add the min version for this property and also for additionalCapabilities

Reply:
Also, pls. add a NOTE that this setting can only be enabled for Host Groups that are created as zonal.

Reply:
@hari-bodicherla I believe the above comment is not addressed in the lates iteration, can you pls. confirm.